.github/workflows/acceptance-tests.yml

```diff
@@ -121,23 +126,22 @@ jobs:
         uses: actions/setup-go@v4
         with:
           go-version-file: 'go.mod'
+      - uses: hashicorp/setup-terraform@v3
+        with:
+          terraform_version: ${{ env.terraform_version }}
```

Comment:
question, why env and not matrix like
https://github.com/mongodb/go-client-mongodb-atlas/blob/d64f483f3c41908a592230ef92ea7912e614d423/.github/workflows/pr.yml#L24-L31

Reply:
this is the first step, it'll surely end with a matrix but I wanted to get closer first and don't break current workflows

Reply:
the matrix will be probably in another workflow calling this one